js/apps/admin-ui/cypress/support/pages/admin-ui/manage/realm_settings/RealmSettingsPage.ts

```diff
@@ -455,10 +457,13 @@ export default class RealmSettingsPage extends CommonPage {
   }
 
   toggleSwitch(switchName: string, waitFor: boolean | undefined = true) {
```

Comment:
Not your code, but this could be simplified to:
```suggestion
  toggleSwitch(switchName: string, waitFor = true) {
```

Reply:
Yup, but I'd rather keep this for future cleanup / refactor

Reply:
Agreed, consider it resolved.